### DISCUSSION TITLE
How do you add a data attributes in a textField function call?

### DISCUSSION THREAD

**Discussion by cfcoder** [wheels-dev/wheels#1349] [Q&A] [answered]
I need to add a data attribute (data-foo-bar) on a text input field. Using the textField function doesn't like any attribute like that, even if I set it equal to true or "true".
I then tried removing the textField function and just inserted the input tag:
`<input id="project-City" maxlength="255" name="project[City]" type="text" value="abc" data-1p-ignore>`
Wheels removes the attribute when the page loads:
`<input id="project-City" maxlength="255" name="project[City]" type="text" value="abc">`

How do others get around this issue?

**Comment by chapmandu** [comment, marked as answer]
You can either wrap the attribute name in quotes, or use underscores. I prefer quoting. 

"data-1p-ignore"=true

or

data_1p_ignore=true

**Reply by cfcoder** [reply]
Thanks for the solution! I should add only the 2nd option worked for me. First one still throws an error.
